.gitignore

```diff
@@ -204,3 +204,14 @@ docs/_build
 *.pb.h
 CMakeFiles*
 build*
+*.cmake
+Makefile
+tags
+
+example_cifar_test
+example_cifar_train
+example_deconv_train
+example_mnist_test
+example_mnist_train
+
+examples/benchmarks_all
```

Comment:
It looks like he's doing in-source builds and adding the temporary build-files into .gitignore to avoid committing them. In-source builds shouldn't be done anyway so these should be removed from .gitignore and @virtan should update his build-setup :-)